src/reanimated2/layoutReanimation/sharedTransitions/SharedTransition.ts

```diff
@@ -14,14 +15,14 @@ import { ReduceMotion } from '../../commonTypes';
 import { configureLayoutAnimations } from '../../core';
 import { ProgressTransitionManager } from './ProgressTransitionManager';
 
-const supportedProps = [
+const SUPPORTED_PROPS = [
```

Comment:
Why do we go for all capital with underscores here? Also, a better approach here would be:
`const myArray = [...] as const`.

Reply:
* We usually use UPPERSCORE for global const. I think that using capital letter shows the purpose of this array better (we emphasise it is some kind of hardcoded SoT)
* Yes `as const` looks better, but it would throw stupid errors, since it doesn't work as we want (it keeps the order inside a type). So the inferred type is not
```
Array<'width' | 'height | ...>
```
It is
```
['width', 'height', ...]
```

Reply:
It is an immutable array - I think it's the right thing. The only pitfall is the need to do `SUPPORTED_PROPS as readonly string[]` instead of `SUPPORTED_PROPS as string[]` but since we cast it anyway I think this is irrelevant.